addons/website/static/tests/tours/custom_popup_snippet.js

```diff
@@ -0,0 +1,35 @@
+/** @odoo-module */
+
+import wTourUtils from "website.tour_utils";
+
+const snippets = [
+    { id: "s_popup", name: "Popup" },
+    { id: "s_popup", name: "Custom Popup" },
+];
+wTourUtils.registerWebsitePreviewTour(
+    "custom_popup_snippet",
+    {
+        test: true,
+        url: "/",
+        edition: true,
+    },
+    [
+        wTourUtils.dragNDrop(snippets[0]),
+        wTourUtils.clickOnSnippet(snippets[0]),
+        {
+            content: "save this snippet to save later",
+            trigger: ".o_we_user_value_widget.fa-save",
+            run: "click",
+        },
+        {
+            content: "confirm and reload custom snippet",
+            trigger: ".modal-footer > .btn.btn-primary",
+            run: "click",
```

Comment:
Hello @panv-odoo In 16.0, "click" is the default action.. So add run: () => {}, // This is a check. here and remove all the run: "click", from the test

Reply:
Hello @panv-odoo,

Thank you for the guidance. I’ll go ahead and remove all instances of run: "click" from the test.

However, I don’t think adding run: () => {}, // This is a check is necessary, as it’s already working fine without it. Let me know if you still want this included, though!

Reply:
Hello @panv-odoo I know its not needed, lets keep it as a check.